modules/cloud-conf-shiftstack-reference.adoc

```diff
@@ -0,0 +1,124 @@
+
+| `domain-name`
+| The Identity service user domain name. 
+
+ This option is not required if you set `domain-id`.
+
+| `tenant-id`
+| The Identity service project ID. Leave this option unset if you are using Identity service application credentials.
+
+In version 3 of the Identity API, which changed the identifier `tenant` to `project`, the value of `tenant-id` is automatically mapped to the project construct in the API. 
+
+| `tenant-name`
+| The Identity service project name. 
+
+
+| `username`
+| The Identity service user name. 
+
+Leave this option unset if you are using Identity service application credentials.
+
+| `password`
+| The Identity service user password. 
+
+ Leave this option unset if you are using Identity service application credentials.
+
+| `region`
+| The Identity service region name.
+
+| `trust-id`
+| The Identity service trust ID. A trust represents the authorization of a user, or trustor, to delegate roles to another user, or trustee. Optionally, a trust authorizes the trustee to impersonate the trustor. You can find available trusts by querying the `/v3/OS-TRUST/trusts` endpoint of the Identity service API.
+|===
+
+[id="ccm-config-lb-options"]
+== Load balancer options
+
+The cloud provider supports several load balancer options for deployments that use Octavia. 
+
+|===
+| Option | Description
+
+| `use-octavia`
+| Whether or not to use Octavia for the `LoadBalancer` type of service implementation rather than Neutron-LBaaS. The default value is `true`.
+
+| `floating-network-id`
+| Optional. The external network used to create floating IP addresses for load balancer virtual IP addresses (VIPs). If there are multiple external networks in the cloud, this option must be set or the user must specify `loadbalancer.openstack.org/floating-network-id` in the service annotation.
+
+
+| `lb-method`
+| The load balancing algorithm used to create the load balancer pool.
+For the Amphora provider the value can be `ROUND_ROBIN`, `LEAST_CONNECTIONS`, or `SOURCE_IP`. The default value is `ROUND_ROBIN`.
+
+For the OVN provider, only the `SOURCE_IP_PORT` algorithm is supported.
+
+For the Amphora provider, if using the `LEAST_CONNECTIONS` or `SOURCE_IP` methods, configure the `create-monitor` option as `true`  in the `cloud-provider-config` config map on the `openshift-config` namespace and `ETP:Local` on the load-balancer type service to allow balancing algorithm enforcement in the client to service endpoint connections.
+
+| `lb-provider`
+| Optional. Used to specify the provider of the load balancer, for example, `amphora` or `octavia`. Only the Amphora and Octavia providers are supported.
+
+| `lb-version`
+| Optional. The load balancer API version. Only `"v2"` is supported.
+
+| `subnet-id`
+| The ID of the Networking service subnet on which load balancer VIPs are created. 
+
+This ID is also used to create pool members if `member-subnet-id` is not set.
```

Comment:
We're mentioning an option that doesn't exist in this version.

Reply:
```suggestion
```

TY!